internal/cmd/users.go

```diff
@@ -353,12 +368,52 @@ func isUserSelf(name string) (bool, error) {
 }
 
 // createUser creates a user with the requested name
-func createUser(client *api.Client, name string, setOTP bool) (*api.CreateUserResponse, error) {
+func createUser(client *api.Client, name string) (*api.CreateUserResponse, error) {
 	logging.S.Debugf("call server: create user named %q", name)
-	user, err := client.CreateUser(&api.CreateUserRequest{Name: name, SetOneTimePassword: setOTP})
+	user, err := client.CreateUser(&api.CreateUserRequest{Name: name})
 	if err != nil {
 		return nil, err
 	}
 
 	return user, nil
 }
+
+// check if the user has permissions to reset passwords for another user.
+// This might be handy for customizing error messages
+func hasAccessToChangePasswordsForOtherUsers(client *api.Client, config *ClientHostConfig) (bool, error) {
+	// TODO: could really use inherited grants for this
+	grants, err := client.ListGrants(api.ListGrantsRequest{
+		User:      config.UserID,
+		Privilege: models.InfraAdminRole,
+		Resource:  "infra",
+	})
+	if err != nil {
+		return false, err
+	}
+
+	if len(grants.Items) > 0 {
+		return true, nil
+	}
+
+	myGroups, err := client.ListGroups(api.ListGroupsRequest{UserID: config.UserID})
+	if err != nil {
+		return false, err
+	}
+
+	for _, group := range myGroups.Items {
+		grants, err := client.ListGrants(api.ListGrantsRequest{
+			Group:     group.ID,
+			Privilege: models.InfraAdminRole,
```

Comment:
This is the CLI referencing models. I believe I've seen a few people suggest in PRs that the CLI should not reference models. I think my suggestion above would remove the need for this.

Reply:
if we eventually separate them we'll have to break these dependencies and/or copy the definitions they point to. Do we want them in two places right now?

Reply:
I think that would be best, or to declare special roles in a mutual package like `api` until we can bring it to the server side